### ISSUE TITLE
running tests on mac big sur shows error message after every run

### ISSUE THREAD

**Issue by van800** [JetBrains/godot-support#63]
![image](https://user-images.githubusercontent.com/1482681/111031178-7145c480-8406-11eb-86d4-007b234af0f8.png)

Temporary workaround https://stackoverflow.com/questions/20226802/disable-the-last-time-you-opened-it-unexpectedly-quit-while-reopening-window#20291771

**Comment by tripodsan** [comment]
for example

```console
$ defaults write -app "/Applications/godot/Godot_mono-3.2.4.rc4.app" NSQuitAlwaysKeepsWindows -bool false
$ rm -rf ~/Library/Saved\ Application\ State/org.godotengine.godot.savedState
$ defaults write com.apple.CrashReporter DialogType none
```

**Comment by van800** [comment]
Fixed in 2021.1.2, which would be released within couple of weeks.
